docs/content/tutorials/firewall.md

```diff
@@ -4,23 +4,7 @@ layout: doc_page
 
 What to Do When You Have a Firewall
 -----------------------------------
-When you are behind a firewall, the Maven Druid dependencies will not be accessible, as well as the IRC wikipedia channels that feed realtime data into Druid. To workaround those two challenges, you will need to:
-
-1. Make the Maven Druid dependencies available offline
-2. Make the Wikipedia example GeoLite DB dependency available offline
-
-## Making Maven Druid Dependencies Available Offline
-1. Extract Druid to a machine that has internet access; e.g. `/Users/foo/druid-<version>`
-2. Create a repository directory to download the dependencies to; e.g. `/Users/foo/druid-<version>\repo`
-3. Create property `druid.extensions.localRepository=`*`path to repo directory`* in the *`Druid Directory`*`\config\_common/common.runtime.properties` file; e.g. `druid.extensions.localRepository=/Users/foo/druid-<version>/repo`
-4. From within Druid directory, run the `pull-deps` command to download all Druid dependencies to the repository specified in the `common.runtime.properties` file:
-
-    ```
-    java -classpath "config\_common;lib\*" io.druid.cli.Main tools pull-deps
-    ```
-
-5. Once all dependencies have been downloaded successfully, replicate the `repo` directory to the machine behind the firewall; e.g. `/opt/druid-<version>/repo`
-6. Create property `druid.extensions.localRepository=`*`path to repo directory`* in the *`Druid Directory`*`/config/_common/common.runtime.properties` file; e.g. `druid.extensions.localRepository=/opt/druid-<version>/repo`
+When you are behind a firewall, the IRC wikipedia channels that feed realtime data into Druid will not be accessible. To workaround this challenge, you will need to make the Wikipedia example GeoLite DB dependency available offline, see below.
```

Comment:
This sentence is a bit confusing, making GeoLite avaible offline will not solve the IRC problem as the sentence implies.
There are two problems, one is that IRC may be firewalled, in which case there's nothing you can do. The other problem is that downloading data from maxmind may not be possible, in which case GeoLite needs to be downloaded offline.